langchain-core/src/utils/js-sha1/hash.ts

```diff
@@ -412,6 +412,19 @@ Sha1.prototype.arrayBuffer = function () {
   return buffer;
 };
 
+/**
+ * @deprecated Use `makeDefaultKeyEncoder()` to create a custom key encoder.
+ * This function will be removed in a future version.
+ */
 export const insecureHash = (message) => {
+  console.warn(
```

Comment:
Let's just log this once on initial call to avoid flooding the console and add some example code

Reply:
Fixed.

Reply:
> add some example code

Can you elaborate on which example code you'd like to be added?